### PR TITLE
Remove bootstrap-icons and include svgs instead

### PR DESCRIPTION
**Pull request by Pelsin**
Replacing bootstrap-icons font files and css with react svg components.

Before:
Skärmavbild 2023-06-13 kl  17 36 21

After:
Skärmavbild 2023-06-13 kl  18 30 10

### CODE REVIEW

File: www/src/Components/ColorScheme.jsx [hunk 1]
```diff
@@ -1,54 +1,63 @@
...
-
-export default ColorScheme;
+import React, { useContext } from "react";
+import { Dropdown } from "react-bootstrap";
+import { AppContext } from "../Contexts/AppContext";
+import SunIcon from "../Icons/Sun";
+import MoonStarsIcon from "../Icons/MoonStars";
+import CircleHalfIcon from "../Icons/CircleHalf";
+
+MoonStarsIcon;
```
Comment (SavageCore) [inline www/src/Components/ColorScheme.jsx:8]: Leftover from debug?

**Review by SavageCore — CHANGES_REQUESTED** [review]
LGTM

File: www/src/Components/ColorScheme.jsx (hunk 1 above)
Comment (Pelsin) [reply www/src/Components/ColorScheme.jsx:8]: Oh you right, Nice!
Thanks for spotting!

**Review by SavageCore — APPROVED** [review]

**Review by arntsonl — APPROVED** [review]
Getting that size down!

**Pull request merged**